presto-hive/pom.xml

```diff
@@ -274,6 +274,12 @@
             <artifactId>units</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.eren-ck</groupId>
+            <artifactId>MortonLib</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
No need to specify the version number here has it has already been specified in the main pom file.